Answers.md

```diff
@@ -1,9 +1,20 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
+1) Refers to what google will look for when scanning your page
+applies to SEO and importance within the web
 
 2. Describe some differences between ```display: block;``` and ```display: inline;```.
+1) Inline will be able to use margin left/right and padding left/right
+while block can use margin, padding, height, and width
```

Comment:
I would add that display block causes a line-break before the next element in the HTML structure.